docs/output.md

```diff
@@ -279,3 +279,18 @@ These curves will be displayed in the pipeline run's MultiQC report, however you
 
 [DamageProfiler](https://github.com/Integrative-Transcriptomics/DamageProfiler)
 is a tool which calculates a variety of standard 'aDNA' metrics from a BAM file. The primary plots here are the misincorporation and length distribution plots. Ancient DNA undergoes depurination and hydrolysis, causing fragmentation of molecules into gradually shorter fragments, and cytosine to thymine deamination damage, that occur on the subsequent single-stranded overhangs at the ends of molecules.
+
+### Contamination estimation for Human DNA
+
+#### ANGSD nuclear contamination
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `contamination_estimation/angsd/`
+
+  - `*.txt`: Text file containing the results of nuclear contamination estimation with ANGSD.
+
+  </details>
+
+[ANGSD](http://www.popgen.dk/angsd/index.php/ANGSD) is a software for analyzing next generation sequencing data. Among other functions, ANGSD can estimate contamination for chromosomes for which one copy exists, i.e. X-chromosome for humans with karyotype XY. To do this, we first generate a binary count file for the X-chromosome (`angsd`) and then perform a Fisher's exact test for finding a p-value and jackknife to get an estimate of contamination (`contamination`).
```

Comment:
Add short explanation of printed contamination table, and explain Method1 vs Method2?